src/lightning/fabric/strategies/fsdp.py

```diff
@@ -127,6 +127,7 @@ def __init__(
         self.cpu_offload = _init_cpu_offload(cpu_offload)
         self.backward_prefetch = backward_prefetch
         self.mixed_precision = mixed_precision
+        self._module: "FullyShardedDataParallel" = ...
```

Comment:
Initializing to ellipsis is not a good idea. Use None instead

Reply:
I wanted to avoid the typing issue here, but yeah will do and add type casts.